aten/src/ATen/function_wrapper.py

```diff
@@ -1563,6 +1563,8 @@ def process_option(option):
         # type: (FunctionOption) -> None
         pair = (backend_type_env['Backend'],
                 backend_type_env['ScalarName'])
+        if pair[1] == 'QInt8':
+            return
```

Comment:
What's going on here?

Reply:
We don't want to generate TH code for QInt8